src/reconcile/caliperEventTsv.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh --
+
+# Send input through stdin or give a filename as an argument
+
+jq -r '[.eventTime[:19]+"Z", .actor.name, .object["@id"], .generated.value//"0"]|@tsv' ${1:--} | \
+    sort | sed 's/%40/@/g'
```

Comment:
This works blazingly fast.  I like the idea of paring down the events to Completed and Skipped via grep first. We are not interested in reconciling other types of events?